custom-login/templates/login.gohtml

```diff
@@ -13,6 +13,7 @@
     display: 'page',
     scope: ['openid', 'profile', 'email'],
```

Comment:
`scope` should be `scopes`

https://github.com/okta/okta-auth-js#authorize-options

Reply:
Thanks @aarongranick-okta !  Fixed

Odd that testing still passed, is scope just deprecated, or removed?